### DISCUSSION TITLE
2024-05-29 CUE Community Update

### DISCUSSION THREAD

**Discussion by myitcv** [cue-lang/cue#3153] [Community Calls]
We are excited to announce our next CUE Community Update for **Wednesday, May 29, 2024 at 1500 UTC.**

Please reply with any questions/topics you would like to see covered.

### Agenda

- Project updates
- Evaluator updates including performance
- CUE with other languages
- Modules and Central Registry updates
- cuelang.org and Docs and Content
- cuepls / LSP updates
- Encodings
- Community updates

### Meeting details

https://meet.google.com/iyx-gqiq-qpt

This session will also be recorded for those unable to join us live.

### CUE Community shared calendar
[Subscribe to the shared calendar](https://calendar.google.com/calendar/u/0?cid=Y19lNzkxMWQ5OWQ4ZGIyMmU2ZTVjMzhkMTVkNjY2ZTVlNjdiNWE5ODNkZWU4N2JmNTU2NDY3NzI1OGIxYjJhMTFhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20).

### Recording

https://youtu.be/N25e0CqR0sk

### Slides

https://docs.google.com/presentation/d/1yn2NLnc20A3FrgOQ_DFHPTwRPEoPAr5OP2hmxOATbLI/edit

### Auto-generated transcript

https://docs.google.com/document/d/1-4QMSyjCZBPBBcWEayU50w_CyF-Fx75NFJQJWf6iDEw/edit

**Comment by antong** [comment]
I think the date in the text is wrong. It says "Tuesday, April 16, 2024 at 1500 UTC".

**Reply by mvdan** [reply]
Thank you for spotting that - fixed.

**Reply by myitcv** [reply]
Apologies for the delay in replying, thanks for catching this @antong!

**Comment by myitcv** [comment]
Updated with recording, transcript and slides
